hugr-py/src/hugr/serialization/ops.py

```diff
@@ -70,42 +70,55 @@ class FuncDecl(BaseOp):
     signature: PolyFuncType
 
 
-CustomConst = Any  # TODO
+class CustomConst(ConfiguredBaseModel):
+    c: str
+
+    @classmethod
+    def set_model_config(cls, config: ConfigDict):
+        cls.model_config = config.copy()
+        cls.model_config["extra"] = "allow"
+
+    class Config:
+        json_schema_extra = {
+            "required": ["c"],
+        }
```

Comment:
I'm a bit confused with `ConfiguredBaseModel`, why do we need this?

I would have expected something like

```python
class CustomConst(BaseMode, extra=Extra.allow):
   c: str
```